### DISCUSSION TITLE
Eco Charge: Support Charge cut-off SOC

### DISCUSSION THREAD

**Discussion by davefx** [mletenay/home-assistant-goodwe-inverter#146] [Ideas]
Hi!

For my ES inverter, the PV Master app supports setting a Charge cut-off SOC when defining a charge period for Eco Mode. 

Could we have that value configurable and available in Home Assistant? I know we could reach that by using an automation, but if this is supported by the Inverter, maybe it should be better to rely on it for it.

**Comment by davefx** [comment]
![imagen](https://user-images.githubusercontent.com/292309/204787857-f6dd0b00-c3b1-450d-864a-2f53e0fcc6ef.png)
This is the field I'm referring to....

**Reply by mletenay** [reply]
Feature added in v0.9.9.9
